controller_lazer_node:
  ros__parameters:
    conv_num: 21
    speed_func: 1
    angle_func: 1
    sleep_msec: 0
